aspnetcore/fundamentals/url-rewriting.md

```diff
@@ -175,30 +221,65 @@ The middleware supports the following Apache mod_rewrite server variables:
+
+[!code-csharp[Main](url-rewriting/samples/1.x/Startup.cs?name=snippet1&highlight=5)]
 
-[!code-csharp[Main](url-rewriting/sample/Startup.cs?name=snippet1&highlight=5)]
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+A `StreamReader` is used to read the rules from the *IISUrlRewrite.xml* rules file.
+
+[!code-csharp[Main](url-rewriting/samples/2.x/Program.cs?name=snippet1&highlight=2,8)]
+
+---
 
-The sample application will rewrite requests from `/iis-rules-rewrite/(.*)` to `/rewritten?id=$1`. The response is sent to the client with a 200 (OK) status code.
+The sample app rewrites requests from `/iis-rules-rewrite/(.*)` to `/rewritten?id=$1`. The response is sent to the client with a 200 (OK) status code.
 
-[!code-xml[Main](url-rewriting/sample/IISUrlRewrite.xml)]
+# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+
+[!code-xml[Main](url-rewriting/samples/1.x/IISUrlRewrite.xml)]
+
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+[!code-xml[Main](url-rewriting/samples/2.x/IISUrlRewrite.xml)]
+
+---
 
 Original Request: `/iis-rules-rewrite/1234`
 
 ![Browser window with Developer Tools tracking the request and response](url-rewriting/_static/add_iis_url_rewrite.png)
 
-If you have an active IIS Rewrite Module with server-level rules configured that would impact your application in undesirable ways, you can disable the IIS Rewrite Module for an application. For more information, see [Disabling IIS modules](xref:hosting/iis-modules#disabling-iis-modules).
+If you have an active IIS Rewrite Module with server-level rules configured that would impact your app in undesirable ways, you can disable the IIS Rewrite Module for an app. For more information, see [Disabling IIS modules](xref:hosting/iis-modules#disabling-iis-modules).
 
 #### Unsupported features
-The middleware does not support the following IIS URL Rewrite Module features:
-* Global rules ([Basic Middleware #169](https://github.com/aspnet/BasicMiddleware/pull/169))
-* Rewrite Maps ([Basic Middleware #168](https://github.com/aspnet/BasicMiddleware/pull/168))
-* CustomResponse action ([Basic Middleware #135](https://github.com/aspnet/BasicMiddleware/issues/135))
-* Custom Server Variables ([Basic Middleware #183](https://github.com/aspnet/BasicMiddleware/issues/183))
-* trackAllCaptures ([Basic Middleware #178](https://github.com/aspnet/BasicMiddleware/pull/178))
+
+# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+
+The middleware released with ASP.NET Core 1.x doesn't support the following IIS URL Rewrite Module features:
+* Global Rules
+* Outbound Rules
+* Rewrite Maps
+* CustomResponse Action
+* Custom Server Variables
 * Wildcards
+* Action:CustomResponse
 * LogRewrittenUrl
 
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+The middleware released with ASP.NET Core 2.x doesn't support the following IIS URL Rewrite Module features:
+* Outbound Rules
+* Custom Server Variables
+* Wildcards
+* Action:CustomResponse
```

Comment:
We actually have support for the CustomResponse action now. https://github.com/aspnet/BasicMiddleware/pull/200